### PR TITLE
Avoid asserting fixed catalog name in TestHiveIntegrationSmokeTest

### PR DESCRIPTION
**Pull request by wenleix**

### CODE REVIEW

**Comment by nezihyigitbasi** [comment]
Please add a short description to the commit message why this change is needed.

File: presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java [hunk 1]
```diff
@@ -1265,17 +1265,17 @@ public void testShowColumnsFromPartitions()
         assertQueryFails(
                 getSession(),
                 "SHOW COLUMNS FROM \"$partitions\"",
-                ".*Table 'hive.tpch.\\$partitions' does not exist");
+                ".*Table '.*.tpch\\.\\$partitions' does not exist");
```
Comment (nezihyigitbasi) [inline presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java]: Why don't you escape the first '.' character, but only the rest: `.*\\.tpch\\.`
You escape it on L1273 below, but not in all changes.

Comment (wenleix) [reply presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java]: The first is not going to be escaped but use `.*` to indicate any match. the rest one is want to use real `.` literal match

Comment (nezihyigitbasi) [reply presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java]: Actually, not the first '.', but the second '.' character. I understand that you are doing literal '.' matches later, but here you use `.*.tpch\\.`, I am asking why not `.*\\.tpch\\.`? On L1273 you use the latter, so there is some inconsistency.

Comment (wenleix) [reply presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java]: @nezihyigitbasi  Good catch!

**Pull request merged**